.github/workflows/test-and-build.yml

```diff
@@ -23,9 +23,8 @@ jobs:
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip
-          pip install -r requirements-airflow.txt -r requirements.txt -r requirements-test.txt
-
+         pip install --no-cache-dir --upgrade setuptools
+         pip install -r requirements-airflow.txt -r requirements.txt -r requirements-test.txt
```

Comment:
🙃 why you have changed it? do we have to update the setuptools? if so, please add `pip install --upgrade pip` as well before

Reply:
yes, we need to update setuptools and wheel, because oterwise inspirehep utils cannot find whee, you can see [here](https://github.com/cern-sis/workflows/runs/6686866925?check_suite_focus=true):
```
      WARNING: The wheel package is not available.
```